### ISSUE TITLE
[Feature Request] Output Device Configuration Selector

### ISSUE THREAD

**Issue by Doekeb** [Rayzeq/quick-settings-audio-panel#58]
Gnome shell version: 46.3.1

Thanks for this amazing extension!

TL;DR: Add a dropdown selector in the panel for the audio output configuration which appears in sound settings.

![image](https://github.com/user-attachments/assets/5d9d58cf-4362-4204-b821-33040bc2c51b)

Context: I have a bluetooth headset that has two configuration modes: "High Fidelity Playback (A2DP Sink)" for high quality stereo listening and "Hands Free Head Unit (HFP)" for lower quality low-latency mono audio. The configuration automatically switches to "Hands Free Head Unit" when the headset is selected as the input device, but does not switch back to "High Fidelity Playback" when I switch back to a different input device. Thus, here's a normal workflow for me:
- Listen to music on "High Fidelity Playback"
- Select headset as audio input to go into a call (this automatically switches output configuration to "Hands Free Head Unit")
- After the call, go into Gnome sound settings and switch configuration back to "High Fidelity Playback."

I would love to be able to do this last step from the panel so I don't have to go all the way into sound settings to change configuration.

**Comment by Rayzeq** [comment]
This is now available in version 63, but the version may not pass the review on the first try, so it may not be available immediately.
The feature will be available for gnome 47 once the gnome 46 version has passed review (I may forget to publish it, if a gnome 47 user wants this feature, they can remind me).
